### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by ue71603**
This is the PR to update the changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,17 @@ All notable changes to this project will be documented in this file. The format
 
 The releases can be found at https://github.com/VDVde/OJP/releases
 
-## [1.1] - Not released yet
+Currently active is [1.0.2]
+
+## [2.0]
+
+### Added
```
Comment (trurlurl) [inline CHANGELOG.md:13]:
```suggestion
### Added
* New 'OJPLineInformationRequest' and 'OJPLineInformationDelivery' ([#243](https://github.com/VDVde/OJP/pull/243), [#308](https://github.com/VDVde/OJP/pull/308))
* New 'OJPAvailabilityRequest' and 'OJPAvailabilityDelivery' ([#262](https://github.com/VDVde/OJP/pull/262))
* Additional 'OptimisationMethod's for trip planning ('environmentalSafety', 'extraSafe', 'extraReliable', 'scenic', 'quietTravel'), new 'HikingProfile' and 'CyclingProfile', support for mutiple 'OptimisationMethod's, new filter 'IncludeAlternativeOptions' to show second-best routes ([#302](https://github.com/VDVde/OJP/pull/302))
* New place sorting options ('PlaceSortingGroup') for 'OJPLocationInformationRequest' ([#301](https://github.com/VDVde/OJP/pull/301))

**Review by trurlurl — COMMENTED** [review]
Updates to V1.1

**Review by trurlurl — COMMENTED** [review]
Another trial to commit from the web editor...

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,7 +6,77 @@ All notable changes to this project will be documented in this file. The format
...
+
+* Renamed `ResultId`s in \*`ResultStructure`s to `Id`, as well as `LegId` in `LegStructure` and `TripId` in `TripStructure`. [#215](https://github.com/VDVde/OJP/pull/215)
+
+
+### Added
+
+* New `OJPLineInformationRequest` and `OJPLineInformationDelivery`. [#243](https://github.com/VDVde/OJP/pull/243), [#308](https://github.com/VDVde/OJP/pull/308)
+* New `OJPAvailabilityRequest` and `OJPAvailabilityDelivery`. [#262](https://github.com/VDVde/OJP/pull/262), [#241](https://github.com/VDVde/OJP/pull/241)
+* New `OJPTripRefineRequest` and `OJPTripRefineDelivery`. [#247](https://github.com/VDVde/OJP/pull/247)
+* New `OJPLineInformationRequest` and `OJPLineInformationDelivery`. [#243](https://github.com/VDVde/OJP/pull/243)
```
Comment (trurlurl) [inline CHANGELOG.md:24]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,7 +6,76 @@ All notable changes to this project will be documented in this file. The format
...
+
+* New OJP.spp (XML Spy project file). [#297](https://github.com/VDVde/OJP/pull/297)
+* Changed location of automatically generated documentation. [#279](https://github.com/VDVde/OJP/pull/279)
+* Migrated from *Travis* to *GitHub Actions*. [#274](https://github.com/VDVde/OJP/pull/274)
+* Renamed `FareProductCodeType` to `FareProductIdType`. [#267](https://github.com/VDVde/OJP/pull/267)
+* Added permalinks to headers in gnerated html documentation. [#246](https://github.com/VDVde/OJP/pull/246)
+* Improved *title* attribute and cardinality in generated HTML documentation. [#233](https://github.com/VDVde/OJP/pull/233)
+
+
+### Changed
```
Comment (trurlurl) [inline CHANGELOG.md:74]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,7 +6,75 @@ All notable changes to this project will be documented in this file. The format
...
+* New OJP.spp (XML Spy project file). [#297](https://github.com/VDVde/OJP/pull/297)
+* Changed location of automatically generated documentation. [#279](https://github.com/VDVde/OJP/pull/279)
+* Migrated from *Travis* to *GitHub Actions*. [#274](https://github.com/VDVde/OJP/pull/274)
+* Renamed `FareProductCodeType` to `FareProductIdType`. [#267](https://github.com/VDVde/OJP/pull/267)
+* Added permalinks to headers in gnerated html documentation. [#246](https://github.com/VDVde/OJP/pull/246)
+* Improved *title* attribute and cardinality in generated HTML documentation. [#233](https://github.com/VDVde/OJP/pull/233)
+
+
+
+### Removed
```
Comment (trurlurl) [inline CHANGELOG.md:75]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,7 +6,74 @@ All notable changes to this project will be documented in this file. The format
...
+* Changed location of automatically generated documentation. [#279](https://github.com/VDVde/OJP/pull/279)
+* Migrated from *Travis* to *GitHub Actions*. [#274](https://github.com/VDVde/OJP/pull/274)
+* Renamed `FareProductCodeType` to `FareProductIdType`. [#267](https://github.com/VDVde/OJP/pull/267)
+* Added permalinks to headers in gnerated html documentation. [#246](https://github.com/VDVde/OJP/pull/246)
+* Improved *title* attribute and cardinality in generated HTML documentation. [#233](https://github.com/VDVde/OJP/pull/233)
+
+
+
+
+## [1.1] - Not released yet (will be released as part of 2.0)
```
Comment (trurlurl) [inline CHANGELOG.md:76]: The following updates should  be added to V1.1:

Added
* Added new files *README.md* and this *CHANGELOG.md*. [#130](https://github.com/VDVde/OJP/pull/130)

Removed
* Removed annotations in `choice` constructs. [#157](https://github.com/VDVde/OJP/pull/157)

**Comment by ue71603** [comment]
@trurlurl Some of the changes should be moved to the category "Breaking", we can also do this later.

File: CHANGELOG.md [hunk 6]
```diff
@@ -6,7 +6,74 @@ All notable changes to this project will be documented in this file. The format
...
+
+### Breaking
+
+* Renamed `ResultId`s in \*`ResultStructure`s to `Id`, as well as `LegId` in `LegStructure` and `TripId` in `TripStructure`. [#215](https://github.com/VDVde/OJP/pull/215)
+
+
+### Added
+
+* New `OJPLineInformationRequest` and `OJPLineInformationDelivery`. [#243](https://github.com/VDVde/OJP/pull/243), [#308](https://github.com/VDVde/OJP/pull/308)
+* New `OJPAvailabilityRequest` and `OJPAvailabilityDelivery`. [#262](https://github.com/VDVde/OJP/pull/262), [#241](https://github.com/VDVde/OJP/pull/241)
```
Comment (trurlurl) [inline CHANGELOG.md:22]:
```suggestion
* New `OJPAvailabilityRequest` and `OJPAvailabilityDelivery`. [#262](https://github.com/VDVde/OJP/pull/262), [#241](https://github.com/VDVde/OJP/pull/241), [#322](https://github.com/VDVde/OJP/pull/322)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -6,7 +6,74 @@ All notable changes to this project will be documented in this file. The format
...
+* New individual mode `park-ride-car`, added `POIAdditionalInformation` for information related to parking or vehicle-sharing. [#280](https://github.com/VDVde/OJP/pull/280)
+* Added `ReservationNeeded` for dated journeys to indicate if and what aspect of a jouney (operation of a service, call at a stop) needs a reservation. [#272](https://github.com/VDVde/OJP/pull/272)
+* Added `SituationFullRefs` for all deliveries. [#268](https://github.com/VDVde/OJP/pull/268)
+* Richer data in `PathGuidance`. [#265](https://github.com/VDVde/OJP/pull/265)
+* Added `UseRealTimeData` switch in `OJPTripRequest` to specify whether and how real-time data is taken into account for the trips returned. [#259](https://github.com/VDVde/OJP/pull/259)
+* Additional enumeration values for `MultiPointType` (`OJPMultiPointTripRequest`), added `ViaSytem` for `OJPTripRequest` and `OJPMultiPointTripRequest`, added `SustainabilityGroup` with `EmissionCO2`, added `Priority` for exchange points, renamed `ReferredSystemId` and `AllowedSystemId` to `ReferredSystem` and `AllowedSystem`, respectively. [#244](https://github.com/VDVde/OJP/pull/244)
```
Comment (trurlurl) [inline CHANGELOG.md:35]:
```suggestion
* Additional enumeration values for `MultiPointType` (`OJPMultiPointTripRequest`), added `ViaSytem` for `OJPTripRequest` and `OJPMultiPointTripRequest`, added `SustainabilityGroup` with `EmissionCO2`, added `Priority` for exchange points, renamed `ReferredSystemId` and `AllowedSystemId` to `ReferredSystem` and `AllowedSystem`, respectively. [#244](https://github.com/VDVde/OJP/pull/244), [#344](https://github.com/VDVde/OJP/pull/344)
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -6,7 +6,74 @@ All notable changes to this project will be documented in this file. The format
...
+* Renamed `ErrorMessage` to `Problem`, `Code` to `Type`, `Text` to `Title` and added `Details` and `LogData`; added dedicated \*`ProblemStructure`s for each request, enumerating the possible types, some of which were previously only defined in the documentation, some not defined at all; added the `Problem` element to the \*`ResultStructure`s. Renamed some problem types, e.g., `FARES_`\* to `FARE_`\*, `MULTIPOINTTRIP_TOOMANYPOINTS` to `TRIP_MULTIPOINT_TOOMANYPOINTS`. [#203](https://github.com/VDVde/OJP/pull/203)
+* Bug fix: renamed `MinimumBookingPeriod.` to `MinimumBookingPeriod`. [#163](https://github.com/VDVde/OJP/pull/163)
+
+
+### Removed
+
+
+### Internal Use
+
+* New OJP.spp (XML Spy project file). [#297](https://github.com/VDVde/OJP/pull/297)
```
Comment (trurlurl) [inline CHANGELOG.md:66]:
```suggestion
* Introduced more specific ObjectIdTypes for NeTEx objects: `FareResultObjectIdType`, `StopEventResultObjectIdType`, `TripResultObjectIdType`, `MultiPointTripResultObjectIdType`, `TripObjectIdType`, `LegObjectIdType`, `PlaceObjectIdType`. [#316](https://github.com/VDVde/OJP/pull/316),  [#348](https://github.com/VDVde/OJP/pull/348)
* New OJP.spp (XML Spy project file). [#297](https://github.com/VDVde/OJP/pull/297)
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -6,7 +6,74 @@ All notable changes to this project will be documented in this file. The format
...
+* Added `AccessibilityFeature`s (values like `stepFreeAccess`, `visualSigns`, etc.) and additional `AccessFeature`s (`singleStep`, `shuttle`, etc.) for `PathLink`s. [#237](https://github.com/VDVde/OJP/pull/237)
+* Heavily extended `BookingArrangements` for better alignment with NeTEx. [#232](https://github.com/VDVde/OJP/pull/232)
+* Added `Operators` in `PlaceResultStructure` and `ResponseContext`, added `IncludeOperators` for `OJPPlaceInformationRequest`. [#220](https://github.com/VDVde/OJP/pull/220)
+* Added `siri:VehicleRef`to `ServiceGroup`. [#214](https://github.com/VDVde/OJP/pull/214)
+* Added `AllowedSystemId`s to `PlaceRefStructure`, specifiying the systems to be queried by `OJPPlaceInformationRequest`. [#204](https://github.com/VDVde/OJP/pull/204)
+* Added `PublicCode` of the journey to `DatedJourney` and `ContinuousService`. [#201](https://github.com/VDVde/OJP/pull/201)
+
+
+### Changed
+
```
Comment (trurlurl) [inline CHANGELOG.md:47]:
```suggestion

* Replaced `TravelClass` with `siri:FareClass` which uses the NeTEx enumeration values. [#354](https://github.com/VDVde/OJP/pull/354)
* Reorganised `BaseTripPolicyGroup` so as to align the parameters in `MultiPointTripPolicyGroup` with `TripPolicyGroup` [#333](https://github.com/VDVde/OJP/pull/333)
* Renamed `OJP_Places.xsd` to `OJP_Location.xsd`, `OJP_PlaceSupport.xsd` to `OJP_LocationSupport.xsd`, `OJPPlaceInformationRequest` to `OJPLocationInformationRequest`, as well as the related Delivery, structures and groups; renamed `Place` to `Location` in the StopEvent service and `TripPlace` to `TripLocation`. [#231](https://github.com/VDVde/OJP/pull/231)
* Updated to SIRI 2.1 using a copy and adapted imports. [#330](https://github.com/VDVde/OJP/pull/330)
* Corrected several typos.  [#320](https://github.com/VDVde/OJP/pull/320),  [#343](https://github.com/VDVde/OJP/pull/343)
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -6,7 +6,74 @@ All notable changes to this project will be documented in this file. The format
...
+## [2.0]
+
+
+### Breaking
+
+* Renamed `ResultId`s in \*`ResultStructure`s to `Id`, as well as `LegId` in `LegStructure` and `TripId` in `TripStructure`. [#215](https://github.com/VDVde/OJP/pull/215)
+
+
+### Added
+
```
Comment (trurlurl) [inline CHANGELOG.md:20]:
```suggestion

* Added `AreaGeometry` to `LineResultStructure`.  [#337](https://github.com/VDVde/OJP/pull/337)
* New `OJPErrorStructure` both at the level of the `xDelivery` and at the level of the `xResult`. [#336](https://github.com/VDVde/OJP/pull/336)
* Added `IsAlternativeOption` to `TripResultStructure`.  [#326](https://github.com/VDVde/OJP/pull/326)
* Added `FareEstimated` to `FareResultStructure`.  [#324](https://github.com/VDVde/OJP/pull/324)
* Added `TripContext` to `TripFareRequestStructure` and `MultiTripFareRequestStructure`.  [#323](https://github.com/VDVde/OJP/pull/323)
* New `OperatorFilter`and `SystemId` in `TripRefineRequest` to indicate the system to be queried. [#298](https://github.com/VDVde/OJP/pull/298)
* Added `ExpectedDepartureOccupancy` and `ExpectedDepartureCapacities` (from SIRI) to Legs.  [#264](https://github.com/VDVde/OJP/pull/264)
* New `OJPStatusRequest`and `OJPStatusDelivery`. [#216](https://github.com/VDVde/OJP/pull/216)
```

**Review by trurlurl — CHANGES_REQUESTED** [review]
Reflects merged PRs as of 24.4.23. Likely to be added next: #353, #447, #349, #363, #328, #341.

File: CHANGELOG.md [hunk 11]
```diff
@@ -6,7 +6,88 @@ All notable changes to this project will be documented in this file. The format
...
+* Renamed `FareProductCodeType` to `FareProductIdType`. [#267](https://github.com/VDVde/OJP/pull/267)
+* Added permalinks to headers in gnerated html documentation. [#246](https://github.com/VDVde/OJP/pull/246)
+* Improved *title* attribute and cardinality in generated HTML documentation. [#233](https://github.com/VDVde/OJP/pull/233)
+
+
+
+
+## [1.1] - Not released yet (will be released as part of 2.0)
 
 ### Added
```
Comment (trurlurl) [inline CHANGELOG.md:93]:
```suggestion

* Added *README.md* and this *CHANGELOG.md* files. [#130](https://github.com/VDVde/OJP/pull/130)
```

File: CHANGELOG.md (hunk 5 above)
Comment (trurlurl) [reply CHANGELOG.md:90]: First item done (#130), still TODO: add line for #157.

File: CHANGELOG.md [hunk 12]
```diff
@@ -6,10 +6,92 @@ All notable changes to this project will be documented in this file. The format
...
+* Added `AllowedSystemId`s to `PlaceRefStructure`, specifiying the systems to be queried by `OJPPlaceInformationRequest`. [#204](https://github.com/VDVde/OJP/pull/204)
+* Added `PublicCode` of the journey to `DatedJourney` and `ContinuousService`. [#201](https://github.com/VDVde/OJP/pull/201)
+
+
+### Changed
+
+* Replaced `TravelClass` with `siri:FareClass` which uses the NeTEx enumeration values. [#354](https://github.com/VDVde/OJP/pull/354)
+* Reorganised `BaseTripPolicyGroup` so as to align the parameters in `MultiPointTripPolicyGroup` with `TripPolicyGroup` [#333](https://github.com/VDVde/OJP/pull/333)
+* Renamed `OJP_Places.xsd` to `OJP_Location.xsd`, `OJP_PlaceSupport.xsd` to `OJP_LocationSupport.xsd`, `OJPPlaceInformationRequest` to `OJPLocationInformationRequest`, as well as the related Delivery, structures and groups; renamed `Place` to `Location` in the StopEvent service and `TripPlace` to `TripLocation`. [#231](https://github.com/VDVde/OJP/pull/231)
+* Updated to SIRI 2.1 using a copy and adapted imports. [#330](https://github.com/VDVde/OJP/pull/330)
```
Comment (trurlurl) [inline CHANGELOG.md:59]:
```suggestion
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -6,10 +6,92 @@ All notable changes to this project will be documented in this file. The format
...
 The releases can be found at https://github.com/VDVde/OJP/releases
 
-## [1.1] - Not released yet
+Currently active is [1.0.2]
+
+## [2.0]
+
+
+### Breaking
+
```
Comment (trurlurl) [inline CHANGELOG.md:15]:
```suggestion

* Updated to SIRI 2.1 using a copy and adapted imports. [#330](https://github.com/VDVde/OJP/pull/330)
```

**Review by trurlurl — CHANGES_REQUESTED** [review]
Moved #330 to *breaking*.

**Comment by ue71603** [comment]
@trurlurl 
The headings should perhaps a bit changed.
I think new service/added/changed/removed would be the titles. The distinction between 1.1 and 2.0 can be removed (as we only have 2.0).
We should mark changes as breaking, when formerly valid examples are no longer valid:
* changes of element names
* changes of structure that are not compatible
* optional elements that are now mandatory
* enum values removed/altered
* type of an element changed (more restrictive/incompatible alteration).
Perhaps we should mark each such line with "[breaking]". 
What do you think?

File: CHANGELOG.md (hunk 5 above)
Comment (trurlurl) [reply CHANGELOG.md]: 157 DONE as well.

**Review by skinkie — APPROVED** [review]

**Comment by trurlurl** [comment]
We are ready, the list is complete & up to date.

**Review by skinkie — APPROVED** [review]

**Comment by ue71603** [comment]
@trurlurl when you accept, we can merge it...

**Pull request merged**